client/src/App.jsx

```diff
@@ -225,6 +253,10 @@ class App extends React.Component {
     }
     return (
       <div>
+        <div id = 'minNightlyRate' style={{display: this.state.minNightlyRate === 'none' ? 'none' : 'block' }}>
```

Comment:
Noticing your use of inline styles. This works, but could be good to review some of the disadvantages.
[https://www.pluralsight.com/guides/react-inline-styling](https://www.pluralsight.com/guides/react-inline-styling)

Reply:
Ok thanks for the note, I'll think about how to refactor this